src/css/override_vars.scss

```diff
@@ -57,6 +57,11 @@ $font-sans:                     '18Franklin-webfont', system, -apple-system, Bli
 $font-serif:                    'Georgia', 'Cambria', 'Times New Roman', 'Times', serif;
 $font-mono:                     'Roboto Mono', 'Bitstream Vera Sans Mono', 'Consolas', 'Courier', monospace;
 
+// utility class scale suffixes
+
+$u-breakpoints: '-xs', '-s', '-m', '-l', '-xl';
+$u-flexscale: '-f1', '-f2', '-f3', '-f4', '-f5', '-f6', '-f7', '-f8', '-f9', '-f10', '-f11';
```

Comment:
I'd prefer readability, even in utility classes, as that's the most common problem I've seen when new developers take over a project.